modules/imgproc/src/sumpixels.avx512_skx.cpp

```diff
@@ -0,0 +1,267 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2019, Intel Corporation, all rights reserved.
+#include "precomp.hpp"
+#include "sumpixels.hpp"
+
+#ifndef CV_ALWAYS_INLINE
+#define CV_ALWAYS_INLINE __attribute__((always_inline))
```

Comment:
This syntax isn't supported by Visual Studio. I agree with @alalek that it would be better to define methods as `static inline`

Reply:
I'm using CLion, so I didn't notice this problem :-(
Switched to static inline - though I like the security of always_inline.

<rant>
Is there no construct that works everywhere for always_inline?
I feel like we need a universal CV_ALWAYS_INLINE at the global scope that will morph the attribute for everyone.

I think all we'd need is 
#ifdef _MSC_VER
#define CV_ALWAYS_INLINE __forceinline
#else 
#define CV_ALWAYS_INLINE __attribute__(always_inline)
#endif 
</rant>
Just need to figure out where to put this and how to test it for every environment.

In the meantime, I just switched it to static inline, but it will run dog slow with BUILD=Debug

Reply:
I've added CV_ALWAYS_INLINE macro in #13617